### PR TITLE
Check pyfiles import properly

### PR DESCRIPTION
**Pull request by jayasimha-raghavan-unskript**
## Description
Please include a summary of the change, motivation and context.

Adding 
* all_module_test.py - This basically uses pythons `importlib` and does the equivalent of `from <CONNECTOR>.legos.<LEGO_DIR>.<LEGO_NAME> import *`. By doing this we achieve two things, 1. Any syntactical error would be caught. 2. Any new package dependency would be caught.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

https://github.com/unskript/Awesome-CloudOps-Automation/actions/runs/5084356176

### Checklist:
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: .github/workflows/all_module_test.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+     
+      - name: Install system dependencies
+        run: |
+          pip install shyaml
+          pip install --upgrade pip==22.2.2
+          sudo apt update
+          sudo apt install -y wget
+          sudo apt install -y awscli
+          aws s3 cp ${{ secrets.BUILD_REQUIREMENTS }} /tmp/requirements.txt
```
Comment (amit-chandak-unskript) [inline .github/workflows/all_module_test.yml:43]: - Need to populate this pip package whenever unskript package is released, probably when the docker release is built.
- Now, if you are adding a new credential, till that credential is released, the legos for that credential will keep failing check. So, we will have to have a new release with the legos check passing.

**Review by amit-chandak-unskript — APPROVED** [review]

**Comment by abhishek-unskript** [comment]
@jayasimha-raghavan-unskript can you take care of the pending review comment so that we can merge this PR.

File: .github/workflows/all_module_test.yml (hunk 1 above)
Comment (jayasimha-raghavan-unskript) [reply .github/workflows/all_module_test.yml:43]: Took care of this.

**Comment by jayasimha-raghavan-unskript** [comment]
@abhishek-unskript the PR is ready to be merged.

**Pull request merged**